### ISSUE TITLE
Importing large opml file doesnt work

### ISSUE THREAD

**Issue by jossephus** [msasikanth/twine#122]
**Describe the bug**
I updated my app version to the latest release and as noted in z changelog this release has support to importing opml files. So I tried to import this opml file [hacker-news-blogs](
https://github.com/outcoldman/hackernews-personal-blogs/blob/master/list.opml). And found these too things to improve

1. Importing only lets us choose xml files at this time... It would be good if opml support was added to
2. Importing z file stops at 8% (can it be because it is large file)

**Device Information**
 - Device: [e.g. Samsung A24]
 - OS: [e.g. Android 13]
 - App Version [e.g. 1.24.0]

**Comment by msasikanth** [comment]
Thanks for reporting the issue. I did try opening an OPML file during development and it worked for me, let me look into it and see if some change caused it to break.

As for the performance, it depends on the size of the OPML. The important factor would be how quickly the app is able to establish a connection with the given links. Sometimes the links in the OPML files might be expired, moved, or taking a while to connect. During my testing, I was able to import a large OPML file (1000+ entries) in 5-7 minutes on my Google Pixel 7. The times are mostly affected by trying to fetch feed & posts while importing the OPML file.

I might revisit that concept later on, but to keep things simple with existing setup, I went with that.
